### DISCUSSION TITLE
Model comparaison

### DISCUSSION THREAD

**Discussion by azertylr** [Akylas/oss-weather#146] [Ideas]
thank for this awesome apps! 

I like the presentation of Weatherian for the temperature+precipitation forecast. Could it be used for the hour per hour forecast or the weather model comparaison?

https://weatherian.com/
![image](https://github.com/Akylas/oss-weather/assets/9157251/10929ec5-4ed4-440e-87b5-01f19aba1f51)

**Comment by farfromrefug** [comment]
@azertylr indeed it look nice. And it is actually really close to what i already show in hour per hour.
Though i dont really want to go for a "graph" view. I prefer to keep it like mobile kind of view. It allows me much more detailed view  for each hour. 
Let s flip the question. What don't you like in  the current hour per hour view? is something bothering you? it is missing details? is it hard to read?

**Reply by azertylr** [reply]
The hour per hour view is very nice and has a lot of details, but you need to scroll several time to the right to get the whole day. On my phone I have can only 5 hours on the screen.
I'd like something more compact (maybe not on the main app page)

**Reply by farfromrefug** [reply]
@azertylr makes sense. Will make some kind of graph view for hourly data. Wont be comparing models though

**Comment by 0-5-0** [comment]
The current hourly view is really nice and does look better than weatherian I find, it shows everything I would need personally.

I had the same kind of thought initially, that it would be nice to see a weatherian like comparison between models directly on the main page however since the design of both apps is very different (weatherian doesn't provide "forecasts"), I sort of realized that it might look odd having the Current forecast report from Open-Meteo ECMWF (rain/no rain), then the Hourly forecast would show all models (rain/no rain) and the Daily forecast would then go back to showing Open-Meteo ECMWF data (rain/no rain).

Having a mix of multiple models on the same page between the 3 forecast types could make it look inaccurate or hard to read, so my thought was it might be more consistent to have the main page show data from only one weather model.

Then @farfromrefug added the comparison view and I was happy :). The "three dots" menu -> Comparison view works nicely to help confirm that the "main" view model is forecasting something sane. I find that useful to help me compare which model might be more accurate as I'm looking outside.

Comparison view might need prettying up though :D, so that it has a nicer design like the hourly view.

**Comment by farfromrefug** [comment]
@azertylr i have been testing that kind of view. What do you think? 
Very basic for now. But in the same time i cant do much more
![Screenshot_1713210682](https://github.com/Akylas/oss-weather/assets/655344/af66202c-3fcf-4ba2-b842-2be514a1838b)

**Reply by azertylr** [reply]
It is pretty neat :) I love it!
just the weather icons that seems a bit small on a smarphone screen?

**Reply by farfromrefug** [reply]
@azertylr indeed it is small. But i cant really make it bigger or they will overlap. It is a question of getting as many "hours" to show while keeping things "readable". not easy
